package.json

```diff
@@ -1,13 +1,11 @@
 {
   "name": "daffodil-debugger",
-  "displayName": "Daffodil Debugger",
+  "displayName": "Apache Daffodil Debugger Extension",
+  "description": "VS Code extension for Apache Daffodil Schema Debugger",
   "version": "1.0.0",
-  "publisher": "jw3",
-  "description": "Daffodil Schema Debugger: debug DFDL schema files using Apache Daffodil.",
+  "publisher": "asf",
+  "author": "Apache Daffodil",
   "license": "Apache-2.0",
```

Comment:
I usually use `Apache License, Version 2.0`, since this is what https://www.apache.org/licenses/LICENSE-2.0 uses.

Reply:
I wasn't aware of spdx, good to know. Apache-2.0 sounds good then.